docs/FAQ/index.md

```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
 Currently the SDL Core repo is built for Ubuntu 18.04 as our default environment.
 
 ## I'm getting a lot of compilation errors, how do I get past them?
-The most common errors come from dependancies issues. Ensure your system has all the required packages to compile the project. Try running the following commands:
+The most common errors come from dependencies issues. Ensure your system has all the required packages to compile the project. Try running the commands in the [dependencies section](../getting-started/install-and-run/#dependencies) of the getting started guide.
 
-```bash
-sudo apt-get update
-sudo apt-get install git cmake build-essential sqlite3 libsqlite3-dev libssl1.0-dev libssl1.0.0 libusb-1.0-0-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools libpulse-dev python3-pip python3-setuptools
+## Can I use SDL on Android OS?
+
+There is no official port at the moment, so individual investigation will need to be done. Even though SDL is designed to work on most Linux systems, modifications might need to be made to the project to get it to work with your setup.
+
+## Why are my RPC requests being DISALLOWED by SDL?
+
+The `DISALLOWED` [result code](https://smartdevicelink.com/en/guides/sdl-overview-guides/rpc-spec/#result) is related to the RPC not being authorized in core's local policy table. Policy permissions for an app are added either in the [preloaded policy table](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json) or through a [policy table update](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/#policy-table-updates).
+
+The [Policies Overview](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/) page provides general information about policies - explaining what they are used for, how the policy table gets updated and how these updates are triggered.[Policy Table Fields](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/policy-fields/) and [App Policies](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/app-policies/) go into more detail about the policy table structure and how to correctly add policy permissions for an application.
```

Comment:
```suggestion
The [Policies Overview](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/) page provides general information about policies - explaining what they are used for, how the policy table gets updated, and how these updates are triggered.[Policy Table Fields](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/policy-fields/) and [App Policies](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/app-policies/) go into more detail about the policy table structure and how to correctly add policy permissions for an application.
```

Reply:
I updated Jack's suggestion to add a space before "[Policy Table Fields]".  This seemed to break showing the actual suggestions, so I've summarized below:
- Add space after period and before "[Policy Table Fields]"
- Add comma after "...how the policy table gets updated"

Reply:
Fixed in https://github.com/smartdevicelink/sdl_core_guides/pull/70/commits/15752fbe4ed9a55cac24c2459a9bb1b85806114f